department/department_app/views/employee_views.py

```diff
@@ -0,0 +1,21 @@
+"""Views employee"""
+from rest_framework import generics
+from department_app.rest.serializers import EmployeeListSerializer
+from department_app.models import Employee
+
+
+class EmployeeCreateView(generics.CreateAPIView):
+    """Create employee"""
+    serializer_class = EmployeeListSerializer
+
+
+class EmployeeListView(generics.ListAPIView):
+    """Read all employee"""
+    serializer_class = EmployeeListSerializer
+    queryset = Employee.objects.all()
+
+
+class EmployeeDetailView(generics.RetrieveUpdateDestroyAPIView):
+    """Read, update, delete"""
+    serializer_class = EmployeeListSerializer
+    queryset = Employee.objects.all()

```

Comment:
Please use ViewSet this will allow to minimize the code.

Reply:
Fixed